Makefile

```diff
@@ -0,0 +1,111 @@
+PROJECT := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+SRC := $(PROJECT)src
+TESTS := $(PROJECT)tests
+ALL := $(SRC) $(TESTS)
+
+export PYTHONPATH = $(PROJECT):$(PROJECT)/lib:$(SRC)
+export PY_COLORS=1
+
+.PHONY: help
+help: ## Show this help.
+	@printf "%-30s %s\n" "Target" "Description"
+	@printf "%-30s %s\n" "------" "-----------"
+	@fgrep " ## " $(MAKEFILE_LIST) | fgrep -v grep | awk -F ': .*## ' '{$$1 = sprintf("%-30s", $$1)} 1'
+
+
+.PHONY: setup-development
+setup-development: ## Install development tools required for development
+	uv add bandit black codespell coverage isort mypy pydocstyle pylint pytest
+	uv add -r requirements.txt
+
+.PHONY: fmt
+fmt: ## Apply coding style standards to code
+	uv run isort $(ALL)
```

Comment:
What do you think about adding a check for uv installed? So user can be warned to run setup-development first.